src/chem/monet_aeronet2ioda.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from monetio import aeronet
+import pandas as pd
+sys.path.append("@SCRIPT_LIB_PATH@")
+
+def aeronet_monet_to_jedi(df, wavelengths=[]):
+    aod_cols = ['aod_{}nm'.format(wv) for wv in wavelengths]
+    cols=aod_cols.copy()
+    # add metadata columns
+    if 'time' not in cols:
+        cols.append('time')
+    if 'latitude' not in cols:
+        cols.append('latitude')
+    if 'longitude' not in cols:
+        cols.append('longitude')
+    if 'elevation' not in cols:
+        cols.append('elevation')
+    if 'aeronet_instrument_number' not in cols:
+        cols.append('aeronet_instrument_number')
+    if 'siteid' not in cols:
+        cols.append('siteid')
+    if df.index.name is not None:
+        index_name = df.index.name
+    else:
+        index_name = 'index'
+
+    q = df.loc[:,~df.columns.duplicated()][cols]
+    w = q.to_xarray().rename({index_name: "x"})
+    nvars = len(aod_cols)
+    w['x'].data = w.x.astype(np.int32)
+    w['nvars'] = arange(1,nvars+1).astype(np.int32)
+    w = w.rename(dict(latitude='latitude@MetaData',
+                      longitude='longitude@MetaData',
+                      time='time@MetaData', x='nlocs',
+                      elevation='height_above_mean_sea_level@MetaData',
+                      aeronet_instrument_number='station_number@MetaData',
+                      siteid='station_id@MetaData'))
+    for index,aod in enumerate(aod_cols):
+        w = w.rename({aod:'aerosol_optical_depth_{}@MetaData'.format(index+1)})
+        w['sensor_channel@VarMetaData'] = (('nvars'), wavelengths)
```

Comment:
This will probably have to be changed soon-ish once IODA can support 2D variables (AOD nchannels x nlocs)